pig/src/test/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusPostBuildAnalyzerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.pnc.bacon.pig.impl.addons.quarkus;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.jboss.pnc.bacon.pig.impl.config.PigConfiguration;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+public class QuarkusPostBuildAnalyzerTest {
+    WireMockServer wireMockServer = new WireMockServer();
+
+    @Test
+    public void analyzePostBuildDeps() throws IOException {
+        QuarkusPostBuildAnalyzer quarkusPostBuildAnalyzer = new QuarkusPostBuildAnalyzer(
+                preConfig(),
+                null,
+                "/test-release-path",
+                "/test-extras-path");
+        wireMockServer.start();
+        stubFor(
+                get(urlEqualTo("/quarkus/?C=M;O=D")).willReturn(
+                        aResponse().withHeader("Content-type", "text/html;charset=UTF-8")
+                                .withBodyFile("build-stub.html")));
+        // stubbing csv files
+        stubFor(
+                get(urlEqualTo("/quarkus/quarkus-1.0.1/extras/community-dependencies.csv")).willReturn(
+                        aResponse().withHeader("Content-type", "text/csv").withBodyFile("community-deps2.csv")));
+        stubFor(
+                get(urlEqualTo("/quarkus/quarkus-1.0.0/extras/community-dependencies.csv")).willReturn(
+                        aResponse().withHeader("Content-type", "text/csv").withBodyFile("community-deps1.csv")));
+        // stubbing text files
+        stubFor(
+                get(urlEqualTo("/quarkus/quarkus-1.0.1/extras/repository-artifact-list.txt")).willReturn(
+                        aResponse().withHeader("Content-type", "text/plain; charset=UTF-8")
+                                .withBodyFile("repository-artifact-list2.txt")));
+        stubFor(
+                get(urlEqualTo("/quarkus/quarkus-1.0.0/extras/repository-artifact-list.txt")).willReturn(
+                        aResponse().withHeader("Content-type", "text/plain; charset=UTF-8")
+                                .withBodyFile("repository-artifact-list1.txt")));
+        quarkusPostBuildAnalyzer.trigger();
+        wireMockServer.stop();
+    }
+
+    @Test
+    public void analyzePostBuildDepsFailTest() throws IOException {
+        QuarkusPostBuildAnalyzer quarkusPostBuildAnalyzer = new QuarkusPostBuildAnalyzer(
+                preConfig(),
+                null,
+                "/test-release-path",
+                "/test-extras-path");
+        quarkusPostBuildAnalyzer.trigger();
```

Comment:
it would be good to check the results of the check too, not only that it's doesn't throw an exception

Reply:
Added verifyResults to check if the results are valid